generated/nifake/nifake/session.py

```diff
@@ -782,6 +782,28 @@ def abort(self):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
+    @ivi_synchronized
+    def accept_list_of_time_values(self, timestamps, delays):
+        r'''accept_list_of_time_values
+
+        Accepts lists of values representing time.
+
+        Args:
+            timestamps (list of float in seconds or datetime.timedelta): A collection of timestamp values.
```

Comment:
I don't know if timestamps should be timedeltas rather than aboslute times?
Do you really need both parameters? Seems like unecessary complexity.

Reply:
Yes, bad naming. I'll use a different variable name.